lusSTR/tests/test_suite.py

```diff
@@ -306,3 +307,16 @@ def test_flank_anno():
         outfile_name = os.path.splitext(outfile.name)[0]
         outfile_name_output = f'{outfile_name}_flanks_anno.txt'
         assert filecmp.cmp(testflanks, outfile_name_output) is True
+
+
+def test_annotate_combine():
+    with NamedTemporaryFile() as outfile:
+        inputfile = data_file('Flanks_testing_file.csv')
+        arglist = ['annotate', inputfile, '-o', outfile.name, '--kit', 'forenseq']
+        args = lusSTR.cli.get_parser().parse_args(arglist)
+        lusSTR.annot.main(args)
+        count=0
+        with open (outfile.name,'rb') as f:
+            for line in f:
+                count+=1
+        assert count == 952
```

Comment:
A "read file all at once" approach would be clearer than a "iterate line by line" approach here in my opinion.

```python
        lusSTR.annot.main(args)
        with open(outfile.name, 'r') as f:
            assert len(f.readlines()) == 952
```

Reply:
Makes sense!